divide_and_conquer/strassen_matrix_multiplication.cpp

```diff
@@ -0,0 +1,457 @@
+#include <cassert>
+#include <chrono>
+#include <iostream>
+#include <tuple>
+#include <vector>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```